flag_test.go

```diff
@@ -2050,3 +2050,34 @@ func TestTimestampFlagApply_WithDestination(t *testing.T) {
 	expect(t, err, nil)
 	expect(t, *fl.Destination.timestamp, expectedResult)
 }
+
+// Test issue #1254
+// StringSlice() with UseShortOptionHandling causes duplicated entries, depending on the ordering of the flags
+func TestSliceShortOptionHandle(t *testing.T) {
+	_ = (&App{
```

Comment:
Nit: Can we add a quick nil error check here to make sure nothing unexpected is happening?

e.g.:

```go
err := (&App{
// ...

if err != nil {
  t.Fatal(err)
}

Reply:
Done.